daml-script/test/BUILD.bazel

```diff
@@ -121,6 +121,7 @@ da_scala_library(
         "@maven//:org_scalatest_scalatest_wordspec",
         "@maven//:org_scalaz_scalaz_core",
     ],
+    visibility = ["//visibility:public"],
```

Comment:
Re-using the helpers in here to test extractor using scripts to initialize the state.